uims_api/exceptions.py

```diff
@@ -1,3 +1,7 @@
 class IncorrectCredentialsError(Exception):
     def __init__(self, message=None):
         super(IncorrectCredentialsError, self).__init__(message)
+
+class UimsFailureError(Exception):
```

Comment:
Let's uppercase `UimsFailureError` to `UIMSFailureError`.

Reply:
Yeah, sure